### PR TITLE
Added example for bad-staticmethod-argument

### PR DESCRIPTION
**Pull request by matusvalo**
Co-authored-by: Vladyslav Krylasov <vladyslav.krylasov@gmail.com>

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Closes partially #5953

### CODE REVIEW

File: doc/data/messages/b/bad-staticmethod-argument/bad.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Klass:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/b/bad-staticmethod-argument/bad.py:1]:
```suggestion
class Wolf:
```

File: doc/data/messages/b/bad-staticmethod-argument/bad.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    @staticmethod
+    def mymethod(self):  # [bad-staticmethod-argument]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/b/bad-staticmethod-argument/bad.py:3]:
```suggestion
    def eat(self):  # [bad-staticmethod-argument]
```

File: doc/data/messages/b/bad-staticmethod-argument/good.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Klass:
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/b/bad-staticmethod-argument/good.py:1]:
```suggestion
class Wolf:
```

File: doc/data/messages/b/bad-staticmethod-argument/good.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+    @staticmethod
+    def mymethod(arg):
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/b/bad-staticmethod-argument/good.py:3]:
```suggestion
    def eat(sheep):
```

**Review by Pierre-Sassoulas — APPROVED** [review]
I have a suggestion to make it more lively :smile:

**Comment by matusvalo** [comment]
@DanielNoord  link in #5953 to this PR is missing.

**Pull request merged**